src/gennylib/include/gennylib/v1/Sleeper.hpp

```diff
@@ -49,6 +50,18 @@ class Sleeper {
     Sleeper(Sleeper&& other) = delete;
     Sleeper& operator=(Sleeper&& other) = delete;
 
+    constexpr void sleep_for(Orchestrator& o, const PhaseNumber pn,
+            const Duration period, bool phaseChangeWakeup = false) const {
+
+        if (phaseChangeWakeup) {
+            // Using locks / condition variables is less efficient/safe, so we
+            // only use this mechanism if the caller explicitly asked for it.
+            o.sleepToPhaseEnd(pn, period);
+        } else if (period.count() && o.currentPhase() == pn) {
```

Comment:
`period.count() > 0`? (Just realized this is the same as elsewhere in this file but it's weird if it's doing the convert-sizes-to-bools thing. Maybe just me.)